rbac/management/group/view.py

```diff
@@ -438,6 +458,10 @@ def update(self, request, *args, **kwargs):
         self.protect_system_groups("update")
 
         group = self.get_object()
+
+        if group.platform_default:
+            self.restrict_custom_default_group_renaming(request)
```

Comment:
Since you get the group above, you can put it into the method. If the name/description different from group.name/description, raise validation error